code/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -198,7 +198,7 @@
     "    --container containers/rna_quantification.sif \\\n",
     "    --reference-fasta reference_data/GRCh38_full_analysis_set_plus_decoy_hla.noALT_noHLA_noDecoy.ERCC.fasta \\\n",
     "    --ref-flat reference_data/Homo_sapiens.GRCh38.103.chr.reformated.ERCC.gtf.ref.flat \\\n",
-    "    --strand unstranded \\\n",
+    "    --strand rf \\\n",
```

Comment:
How do you know that the MWE is `rf` -- can we show people how this is determined, as part of the documentation?